### PR TITLE
Update Docs For Monorepo support

### PR DESCRIPTION
**Pull request by InTheCloudDan**
Add basic info around monrepo support.

### CODE REVIEW

**Review by jazanne — APPROVED** [review]

File: docs/CONFIGURATION.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The following arguments are required to run `ld-find-code-refs`. For a detailed
...
 | `accessToken` | LaunchDarkly [personal access token](https://docs.launchdarkly.com/docs/api-access-tokens) with writer-level access, or access to the `code-reference-repository` [custom role](https://docs.launchdarkly.com/v2.0/docs/custom-roles) resource |
 | `dir`         | Path to existing checkout of the git repo. The currently checked out branch will be scanned for code references.                                                                                                                               |
-| `projKey`     | A LaunchDarkly project key.                                                                                                                                                                                                                    |
+| `projKey`     | A LaunchDarkly project key. Cannot be used with `projects` block.                                                                                                                                                                                                                   |
```
Comment (yusinto) [inline docs/CONFIGURATION.md:19]: The intent here is to make clear that either projKey or projects is required, so maybe we can improve this readme like so:
```
| `projKey` or `projects` | A LaunchDarkly project key or a projects block containing a list of project keys. See Advanced configuration for more details.                                                                                               |
```
What do you think?

File: docs/CONFIGURATION.md [hunk 2]
```diff
@@ -96,12 +96,25 @@ contextLines: 3
...
+#### Projects
+
+Projects allow Code References to scan a monorepo for multiple LaunchDarkly Projects in a single run. If a `dir` is provided Code References will only start looking for that Project's feature flag keys and aliases below that directory.
+
+Each Project block can support an optional list of alias configuration blocks. Any globally defined aliases will be inherited.
+
+```yaml
+projects:
+    - key: example-project
+      dir: subdir/to/start
```
Comment (yusinto) [inline docs/CONFIGURATION.md:114]: Nit: does it make more sense to say `subRepo/to/scan` here?
```
dir: subRepo/to/scan
```

**Review by yusinto — APPROVED** [review]

**Comment by yusinto** [comment]
To re-surface my suggestion from our private conversation, if it's not too much work it will be beneficial and more complete for our customers if there's an example for monorepos in docs/EXAMPLES.md as well.

**Comment by InTheCloudDan** [comment]
Re-opening as a new PR because it was not merged correctly in the stacked PR

**Pull request closed without merging**